monai/transforms/io/array.py

```diff
@@ -315,11 +316,14 @@ class SaveImage(Transform):
 
     Args:
         output_dir: output image directory.
+        Handled by ``folder_layout`` instead, if ``folder_layout`` not ``None``.
```

Comment:
if ``folder_layout`` **is** not ``None``.
Same for other places.

Thanks.

Reply:
Will fix